src/main/java/org/apache/commons/lang3/builder/ToStringStyle.java

```diff
@@ -386,7 +389,7 @@ public void appendSuper(final StringBuffer buffer, final String superToString) {
      */
     public void appendToString(final StringBuffer buffer, final String toString) {
         if (toString != null) {
-            final int pos1 = toString.indexOf(contentStart) + contentStart.length();
+            @SuppressWarnings("assignment.type.incompatible") final @LTEqLengthOf("toString") int pos1 = toString.indexOf(contentStart) + contentStart.length(); // maximum value of toString.indexOf(String str) + str.length() can be toString.length()
```

Comment:
Since `@SuppressWarnings` is a declaration annotation, it's usually considered better style to put it on a separate line.

Reply:
(that also makes it easier to read)